zipline/data/data_portal.py

```diff
@@ -703,10 +697,17 @@ def _get_history_daily_window_data(
 
             return daily_data
 
-    def _handle_history_out_of_bounds(self, bar_count):
+    def _handle_minute_history_out_of_bounds(self, bar_count):
+        first_trading_minute_loc = (
```

Comment:
I thought about caching this like we were doing before, but we're only calculating this offset so that we can give a useful error message in an out of bounds exception.  We don't expect this to be needed again.  The only case where this would help performance would be someone who's making invalid history calls in a loop.  Do you think it's still worth having the lazyval?